helix-term/src/commands.rs

```diff
@@ -5620,7 +5705,7 @@ fn add_newline_impl(cx: &mut Context, open: Open) {
 }
 
 fn rename_symbol(cx: &mut Context) {
-    let prompt = Prompt::new(
+    let promt = Prompt::new(
```

Comment:
This should be `prompt` ?

Reply:
Whoops yeah thats just a left over from when I tested the rename symbol command :)